spec/class_room.spec.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../class_room'
+
+describe Classroom do
+  context 'testing the Classroom class' do
+    it 'create a new classroom when Classroom class is initialised' do
+      cl = Classroom.new('1E')
+      expect(cl.label).to eq '1E'
+    end
+  end
+end
```

Comment:
- [ Required ] Please consider adding unit tests for the following methods in the `Classroom` class. It is part of the project requirements that all methods in a class are tested.
 
  - `add_student(student)`